frontend/postcss.config.js

```diff
@@ -3,11 +3,11 @@ import postcssViewportToContainerToggle from 'postcss-viewport-to-container-togg
 export default {
   // Is the site still editable in production like a normal Apos Site,
   // if yes we need the plugin for all builds
-  plugins: [
-    postcssViewportToContainerToggle({
-        modifierAttr: 'data-breakpoint-preview-mode',
-        debug: true,
-        transform: null
-    })
-  ]
+  // plugins: [
+  //   postcssViewportToContainerToggle({
+  //       modifierAttr: 'data-breakpoint-preview-mode',
+  //       debug: true,
+  //       transform: null
+  //   })
+  // ]
```

Comment:
didn't you just do something w this?

Reply:
I added a lot of functionality so it is working better for straight Apostrophe projects, but for some reason it is like Astro is running the CSS through the build process twice, leading to mangled selectors.